### PR TITLE
Address react 15.5 deprecations regarding PropTypes usage and test-utils

### LINKED ISSUE #35: Address react 15.5 deprecations regarding PropTypes usage and test-utils
## description
Check react 15.5 release blog posts regarding this issue:
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

## how to reproduce
- {optional: describe steps to reproduce defect}

## specs
{optional: describe a possible fix for this defect, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by patresi**
Closes #35 

Check react 15.5 release blog posts regarding this issue:
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
Thank you for this PR @patresi !

File: package.json [hunk 1]
```diff
@@ -61,16 +61,17 @@
     "karma-mocha": "^0.2.1",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
-    "react": "^15.4",
+    "react": "^15.5",
     "react-addons-test-utils": "^0.14.8 || ^15.0.0",
```
Comment (FrancescoCioria) [inline package.json]: `react-addons-test-utils` is not used anymore right? if yes, can you remove it from `package.json`?

Comment (patresi) [inline package.json]: done :)

**Comment by gabro** [comment]
@FrancescoCioria CI is failing for a silly reason, you can merge ;)

**Comment by FrancescoCioria** [comment]
> @FrancescoCioria CI is failing for a silly reason, you can merge ;)

yeah I know, it doesn't handle correctly PRs from forked repos

**Pull request merged**